ivy/functional/backends/tensorflow/experimental/losses.py

```diff
@@ -1,9 +1,32 @@
 import tensorflow as tf
 from typing import Optional
+import ivy
 from ivy.func_wrapper import with_unsupported_dtypes
 from . import backend_version
 
 
+@with_unsupported_dtypes({"2.13.0 and below": "bool"}, backend_version)
+def smooth_l1_loss(
```

Comment:
Don't forget to modify the name of this function, I see you've implemented huber_loss here